addons/web/static/src/core/assets.js

```diff
@@ -179,6 +215,7 @@ export const loadPublicAsset = memoize(async function loadPublicAsset(xmlid, orm
         jsLibs: [...doc.querySelectorAll("script[src]")].map((node) => node.getAttribute("src")),
     });
 });
+
```

Comment:
Are these additions of a bunch of empty lines nowhere near the edited content really necessary?